woocommerce/woocommerce-ios
# [Auth] Remove the `.show2FA` segue and navigate programmatically

Ref. #2003 

### In this PR
- Authenticator has been updated to remove the storyboard segue `.show2FA`
- See also wordpress-mobile/WordPressAuthenticator-iOS#228

### To Test
0. Enable [2-factor authentication](https://wordpress.com/support/security/two-step-authentication/) for your test site
1. Check out this branch
2. `rake dependencies`
3. Run unit tests
4. Build and run
5. Log out
6. Select the "Log in with Jetpack" > use email address > enter password instead > Next
**Expected result:** it should show the 2-factor authentication screen:

<img src="https://user-images.githubusercontent.com/1062444/78177480-d128a280-7423-11ea-8f65-92bc8d0166a0.png" width="350" />

### Do not merge until
1. The Authenticator PR has been merged: wordpress-mobile/WordPressAuthenticator-iOS#228
2. A new Authenticator release has been created 
3. The release is published on Cocoapods trunk 
4. This PR points to the new release 

Update release notes:
